tableauserverclient/models/data_alert_item.py

```diff
@@ -9,44 +9,50 @@
 from .view_item import ViewItem
 
 
+from typing import List, Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from datetime import datetime
+
+
 class DataAlertItem(object):
     class Frequency:
-        Once = "Once"
-        Frequently = "Frequently"
-        Hourly = "Hourly"
-        Daily = "Daily"
-        Weekly = "Weekly"
+        Once: str = "Once"
```

Comment:
I think these enum-like classes don't really need the type hint, it's pretty obvious it's a string and isn't going to change. Mypy is smart enough to infer from the literal.

Reply:
Removed type hints from the frequency namespace